docs/src/man/reshaping_and_pivoting.md

```diff
@@ -53,29 +53,6 @@ If the remaining columns are unique, you can skip the id variable and use:
 widedt = unstack(longdt, :variable, :value)
 ```
 
-`stackdt` and `meltdt` are two additional functions that work like `stack` and `melt`, but they provide a view into the original wide DataTable. Here is an example:
-
-```julia
-d = stackdt(iris)
-```
-
-This saves memory. To create the view, several AbstractVectors are defined:
-
-`:variable` column -- `EachRepeatedVector`  
```

Comment:
There aren't any `EachRepeatedVector`s left in the code so this is probably overdue for removal